### ISSUE TITLE
Extração do login de dentro da mensagem provavelmente não tá funcionando

### ISSUE THREAD

**Issue by manoelcampos** [Gustaprof25/Blocking-chat#3]
A variável user deve conter o login digitado na msg.
A linha abaixo não parece estar fazendo isto, use um print pra confirmar.
Como vc agrupou as partes da msg (usando parenteses), e o login está representado pela regex dentro do segundo parenteses, usando matcher.groups(2) deve extrair o conteúdo desse segundo grupo e armazenar na variável user.
Assim, vc saberá pra qual usuário é a mensagem.

Depois é imprimir uso variável pra confimar se o login foi obtido da msg 

https://github.com/Gustaprof25/Blocking-chat/blob/3f09151383a882758629b9a8fb5f94607e349009/1.1-blocking-socket/src/main/java/com/manoelcampos/chat/BlockingChatServer.java#L161

**Comment by Gustaprof25** [comment]
Professor já confirmei! A variável user  contem o nome do usuário. E eu não consegui obter usando o matcher.group()

**Comment by manoelcampos** [comment]
Certo, mas o problema anterior que citei não foi corrigido. O código continua do mesmo jeito. Veja issue #2, exatamente [esta mensagem](https://github.com/Gustaprof25/Blocking-chat/issues/2#issuecomment-839240920).
